src/main/java/com/odysseusinc/arachne/portal/config/WebSecurityConfig.java

```diff
@@ -309,11 +323,22 @@ protected void configure(HttpSecurity http) throws Exception {
                 .anyRequest().permitAll();
 
         // Custom JWT based authentication
-        http.addFilterBefore(loginRequestFilter(), UsernamePasswordAuthenticationFilter.class);
+        http.addFilterBefore(loginRequestFilter(), OAuth2AuthorizationRequestRedirectFilter.class);
         http.addFilterBefore(authenticationTokenFilterBean(), LoginRequestFilter.class);
         // DataNode authentication
         http.addFilterBefore(authenticationSystemTokenFilter(), AuthenticationTokenFilter.class);
         http.addFilterBefore(hostfilter, AuthenticationSystemTokenFilter.class);
+//        http.addFilterAfter(oauthUserProviderFilter, SessionManagementFilter.class);
+
+    }
+
+    private OidcUserService oidcUserService() {
+        Set<String> scopes = Stream.of(
+                "openid", "profile", "email", "address"
```

Comment:
Probably, it's better to move scopes into configuration options

Reply:
I humbly disagree. We have hard code dependency on all of these scopes, moving them to configuration does not yield any benefit and creates more space for mistakes.